### ISSUE TITLE
Object of type "Callable" is not callable inside a Tuple

### ISSUE THREAD

**Issue by asfaltboy** [microsoft/pyright#1675]
**Describe the bug**

When I define a type alias as a `Tuple` with a `Callable` element. Calling the function in some bit of code is reported as an error:

    Object of type "Callable" is not callable

On the other hand, a `'Callable'` type inside a `NamedTuple` is not reported back as an issue.

**To Reproduce**

1. Download https://gist.github.com/asfaltboy/26a0e5a2372f2a3316ae874fc3494e62
2. Run `pyright callable_bug.py`

**Expected behavior**

I expect no errors to be returned, whether using `Tuple` or `NamedTuple`.

**Screenshots or Code**

```python
# No error here
# class RuleDefinition(NamedTuple):
#     pattern: str
#     path: str
#     check: Callable

# error: Object of type "Callable" is not callable
RuleDefinition = Tuple[str, str, Callable]

def check_rule(rule: RuleDefinition) -> str:
    pattern, path, check = rule
    if check(pattern=pattern, path=path):
        return path
    return ""
```

**VS Code extension or command-line**

Using CLI, version 1.1.123, freshly installed with npm.

**Additional context**

See attached file to reproduce.
Mypy returns no errors in both cases

**Comment by erictraut** [comment]
Thanks for the bug report. 

You have omitted the type arguments for `Callable`, which is generally a bad idea. That said, PEP 484 indicates that a "bare" `Callable` is permitted and should be treated as `Callable[..., Any]`. Pyright will translate it to `(*args: Unknown, **kwargs: Unknown) -> Unknown`. 

This will be addressed in the next release of pyright. In the meantime, you can avoid the bug by specifying proper type arguments for `Callable`.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.124, which I just published. It will also be included in the next release of pylance.
